<?xml version="1.0"?>
<launch>
    <arg name="detector"            default="simple"    doc="simple" />
    <arg name="use_lidar_center"    default="true"      doc="whether to use center lidar for detection" />

    <group ns="detection">
        <group if="$(arg use_lidar_center)" ns="lidar_center">
            <remap from="points_raw" to="/points_raw_center" />
            <include if="$(eval arg('detector')=='simple')" file="$(find autoware_mini)/launch/detection/lidar_simple.launch" />
        </group>

        <rosparam command="load" file="$(find autoware_mini)/config/detection.yaml"/>
    </group>

</launch>